examples/environment-guide/containers/show/show.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2015 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
+)
+
+func getKubeEnv() (map[string]string, error) {
+	environS := os.Environ()
+	environ := make(map[string]string)
+	for _, val := range environS {
+		split := strings.Split(val, "=")
+		if len(split) != 2 {
+			return environ, fmt.Errorf("Some weird env vars")
+		}
+		environ[split[0]] = split[1]
+	}
+	for key := range environ {
+		if !(strings.HasSuffix(key, "_SERVICE_HOST") ||
+			strings.HasSuffix(key, "_SERVICE_PORT")) {
```

Comment:
Ah, I see. That should probably be in https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/container-environment.md as well. I'll file an issue.